### PR TITLE
Add service update error management

### PR DESCRIPTION
**Pull request by robyoung**
This adds service update error management to the content loader module.

These methods are copied over from the supplier frontend and will be used both from there and the admin frontend.

### CODE REVIEW

File: tests/test_content_loader.py [hunk 1]
```diff
@@ -628,6 +628,123 @@ def test_get_field_names_with_no_pricing_question(self):
...
+        section = ContentSection.create({
+            "id": "second_section",
+            "name": "Second section",
+            "questions": [{
+                "id": "q2",
+                "question": "Second question",
+                "type": "text",
+            }]
+        })
+        assert section.has_changes_to_save({'q2': 'foo'}, {'q2': 'food'})
```
Comment (quis) [inline tests/test_content_loader.py]: `foo` and `food` is a pretty subtle difference scanning through the code!

**Comment by quis** [comment]
Apart from that one comment, this looks good.

**Comment by quis** [comment]
:+1:

File: dmutils/content_loader.py [hunk 2]
```diff
@@ -202,6 +202,77 @@ def get_data(self, form_data):
...
+                return 'no_min_price_specified'
+            elif question_id == 'priceUnit':
+                return 'no_unit_specified'
+        elif message_key == 'not_money_format':
+            if question_id == 'priceMin':
+                return 'min_price_not_a_number'
+            elif question_id == 'priceMax':
+                return 'max_price_not_a_number'
+        return message_key
+
```
Comment (tommorris) [inline dmutils/content_loader.py]: Is `_rewrite_pricing_error_key` still needed?

**Pull request merged**